atd-vze/src/views/Crashes/UnitDetailsCard.js

```diff
@@ -16,22 +16,12 @@ import { unitDataMap } from "./unitDataMap";
 import { GET_UNIT_LOOKUPS } from "../../queries/lookups";
 import { GET_UNITS, UPDATE_UNIT } from "../../queries/units";
```

Comment:
this is also out of scope for this work—but we're going to want to update the related records cards so that they refetch the crash query whenever data is edited, so that the metric cards and change logs refresh accordingly. i'll capture that in our todos.